sycl/include/sycl/ext/intel/experimental/esimd/math.hpp

```diff
@@ -198,14 +273,29 @@ esimd_shr(T1 src0, T2 src1, int flag = saturation_off) {
   return Result[0];
 }
 
-// esimd_rol
+/// Rotate left operation (vector version)
+/// \tparam T0 element type of the returned vector. Must be any integer type.
+/// \tparam T1 element type of the input vector. Must be any integer type.
+/// \tparam SZ size of the input and returned vectors.
+/// @param src0 the input vector.
+/// @param src1 the vector with number of bit positions by which the elements of
+/// the input vector \p src0 shall be rotated.
+/// @return vector of rotated elements.
 template <typename T0, typename T1, int SZ>
 ESIMD_NODEBUG ESIMD_INLINE typename sycl::detail::enable_if_t<
     std::is_integral<T0>::value && std::is_integral<T1>::value, simd<T0, SZ>>
 esimd_rol(simd<T1, SZ> src0, simd<T1, SZ> src1) {
   return __esimd_rol<T0, T1, SZ>(src0, src1);
 }
 
+/// Rotate left operation (vector version)
```

Comment:
(here and if few more places) Description matches the previous function, even though they are different. Please update.

Reply:
Done